komga-webui/src/locales/en.json

```diff
@@ -51,24 +51,24 @@
     "button_scan": "Scan",
     "button_select_series": "Select Series",
     "field_import_path": "Import from folder",
-    "info_part1": "This screen lets you import files that are outside your existing libraries. You can only import files into existing Series, in which case Komga will move or copy the files into the directory of the chosen Series.",
-    "info_part2": "If you choose a number for a book, and a book already exists with that number, then you will be able to compare the 2 books. If you decide to import the book, Komga will upgrade the existing book with the new one, effectively replacing the old file with the new.",
+    "info_part1": "Here you can import files into a selected series of your existing libraries.",
+    "info_part2": "Choose a number for a book and compare the two if it exists. Importing the book upgrades the entry and replaces the old file.",
     "no_files_found": "No files found",
     "notification": {
       "go_to_book": "Go to book",
-      "import_failure": "Failed to import book: {file}",
-      "import_successful": "Book imported successfully: {book}",
+      "import_failure": "Could not import the {file} book",
+      "import_successful": "Imported {book}",
       "source_file": "Source file: {file}"
     },
     "row": {
-      "error_analyze_first": "Book needs to be analyzed first",
```

Comment:
Marked noun error.

Reply:
> Marked noun error.

What does that mean ?

---

Your form seems to indicate that this is a user action, while the existing one implies, to me at least, that this is still ongoing and will happen at some point.

Reply:
It can either be "unanalyzed book" or "The book …"

Reply:
i am sorry but i don't understand what you are saying

Reply:
Man no luck explaining, but tries.

Reply:
https://learnenglish.britishcouncil.org/grammar/english-grammar-reference/the-definite-article-the
Not marking the noun leaves it undefined.
Is it specifically "a book", "the book" or "some books".

Reply:
Fair enough, thanks for taking the time to explain this.